src/clj/game/cards/resources.clj

```diff
@@ -1455,7 +1468,14 @@
                                                     (system-msg state :runner (str "places 1 [Credits] on Net Mercur"))
                                                     (add-counter state :runner card :credit 1)
                                                     (effect-completed state side eid))))}
-                                card nil))}}}
+                                card nil))}}
+    ; Normally this should be (req true), but having pay-credits prompts on
```

Comment:
I disagree strongly. It's an extra click, which is annoying, but just yesterday at my Regionals, I lost because the runner beat my Punitive Trace by spending all of their Net Mercur credits. (This isn't the best example because Trace prompts won't work with this system at the moment, lol, but you get my meaning.)

The card says "anything", so I think it should be for anything. Maybe let this be toggle-able somehow? Give Net Mercur an ability that changes whether you want it to ask or not?

Reply:
For now you can simply revert to the old system of clicking on Net Mercur to get credits. That ability hasn't changed.

But I like the idea of having it toggle-able!